.lintstagedrc.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+
+const buildEslintCommand = (filenames) =>
+	`yarn next:lint --fix --file ${filenames
+		.map((f) => path.relative(path.join('packages','frontend'), f))
+		.join(' --file ')}`;
+
+module.exports = {
+	'packages/frontend/**/*.{ts,tsx}': [buildEslintCommand],
+};
```

Comment:
This has a weird formatting. I think its using tabs instead of space.

Also not following the prettier rules. (double quote, etc)